spec.md

```diff
@@ -2961,6 +2961,104 @@ For this reason, it is an error to use `rwprobe`{.firrtl} on any port on a publi
 
 Probes may not be inputs to modules.
 
+# Contracts
+
+The `contract`{.firrtl} declaration provides a mechanism to verify a predicate or property, and to then use this predicate or property to simplify the circuit for further verification or simulation.
```

Comment:
Supreme-nit: please use two sentences here to make this easier to read.

Reply:
also nit didn't we have some sort of one-sentence-per-line rule at some point?